### PR TITLE
280 multiple operators

### PR DESCRIPTION
**Pull request by marcellmueller**
Implements #280 

Note: currently if you save the form with multiple operators, then remove them and save the form again they will still appear in the form data. I have created issue #361 to handle this as it is blocked on #326 to implement.

This includes major changes to the `operators` form including moving the multiple operators section to page 1 as in the Figma designs. The multiple operators form is revealed if you choose `Yes` for the question `Does the operation have multiple operators?` at the bottom of page 1. You can add as many operators as you want if you click the `Add another operator` button.

Screenshot 2023-12-05 at 1 57 48 PM

### CODE REVIEW

**Review by dleard — CHANGES_REQUESTED** [review]

File: bc_obps/registration/models.py [hunk 1]
```diff
@@ -491,3 +493,77 @@ class Meta:
...
+        max_length=1000,
+        db_comment="The physical municipality of an operator ",
+    )
+    physical_province = CAProvinceField(
+        db_comment="The physical street address of an operator, restricted to two-letter province postal abbreviations"
+    )
+    physical_postal_code = CAPostalCodeField(
+        db_comment="The physical postal code address of an operator, limited to valid Canadian postal codes"
+    )
+    mailing_address_same_as_physical = models.BooleanField(
```
[the remark this answers is not in the document]
Comment (marcellmueller) [reply bc_obps/registration/models.py:555]: These actually do need to be a column in the table or else the form displays incorrectly as this data is lost after saving and reloading.

File: bc_obps/registration/migrations/0001_initial.py [hunk 2]
```diff
@@ -522,6 +528,108 @@ class Migration(migrations.Migration):
...
+                (
+                    'mailing_postal_code',
+                    localflavor.ca.models.CAPostalCodeField(
+                        db_comment='The mailing postal code of an operator, limited to valid Canadian postal codes',
+                        max_length=7,
+                    ),
+                ),
+                (
+                    'operation',
+                    models.ForeignKey(
```
[the remark this answers is not in the document]
Comment (marcellmueller) [reply bc_obps/registration/migrations/0001_initial.py:621]: Thanks, updated!

File: bc_obps/registration/models.py [hunk 3]
```diff
@@ -491,3 +493,77 @@ class Meta:
...
+        related_name="multiple_operator",
+    )
+    operator_number = models.IntegerField(
+        db_comment="Number used to differentiate operators for this operation for saving/updating purposes"
+    )
+    legal_name = models.CharField(max_length=1000, db_comment="The legal name of an operator")
+    trade_name = models.CharField(max_length=1000, db_comment="The trade name of an operator")
+    cra_business_number = models.IntegerField(db_comment="The CRA business number of an operator")
+    bc_corporate_registry_number = models.IntegerField(db_comment="The BC corporate registry number of an operator")
+    business_structure = models.CharField(
```
[the remark this answers is not in the document]
Comment (marcellmueller) [reply bc_obps/registration/models.py:514]: Ah yeah I should have wrote a comment about this. At the time I was working on this PR the business structure functionality for the user operator form was being worked on in another PR so I left this. 

I started to update this though I think I should do it in a separate PR to make it easier to review.

File: client/app/components/form/OperationsForm.tsx [hunk 4]
```diff
@@ -108,6 +109,7 @@ export default function OperationsForm({ formData, schema }: Props) {
               operator_id: 1,
```
[the remark this answers is not in the document]
Comment (marcellmueller) [reply client/app/components/form/OperationsForm.tsx:109]: It needs to be there for the form to submit. How do I go about getting the operator id?

File: bc_obps/registration/models.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (dleard) [reply bc_obps/registration/models.py:556]: Ah, yeah makes sense

**Review by dleard — APPROVED** [review]
Looks like it needs a rebase, but otherwise good to go (unless you're going to tackle the change to 'business_structure' in this PR rather than a subsequent one)

**Pull request merged**